site/_docs/reference.md

```diff
@@ -2812,6 +2812,7 @@ In the following:
 | b m p s | MD5(string)                              | Calculates an MD5 128-bit checksum of *string* and returns it as a hex string
 | m | MONTHNAME(date)                                | Returns the name, in the connection's locale, of the month in *datetime*; for example, it returns '二月' for both DATE '2020-02-10' and TIMESTAMP '2020-02-10 10:10:10'
 | o s | NVL(value1, value2)                          | Returns *value1* if *value1* is not null, otherwise *value2*
+| s | NVL2(value1, value2, value3)                   | Returns *value2* if *value1* is not null, or *value3* otherwise.
```

Comment:
remove trailing '.'

Oracle and Spark

Why ', or value3 otherwise' and not ', otherwise value3' when line above says 'otherwise value2'?

Reply:
fix